test/00_operators/OperatorTests.cu

```diff
@@ -2378,6 +2378,84 @@ TYPED_TEST(OperatorTestsNumericAllExecs, Reshape)
   MATX_EXIT_HANDLER();
 }
 
+TYPED_TEST(OperatorTestsNumericNonComplexAllExecs, Overlap)
+{
+  MATX_ENTER_HANDLER();
+
+  using TestType = std::tuple_element_t<0, TypeParam>;
+  using ExecType = std::tuple_element_t<1, TypeParam>;
+  using inner_type = typename inner_op_type_t<TestType>::type;
+
+  ExecType exec{}; 
+
+
+  tensor_t<TestType, 1> a{{10}};
+  a.SetVals({0, 1, 2, 3, 4, 5, 6, 7, 8, 9});
+  auto ao = overlap(a, {4}, {2});
```

Comment:
Perhaps also have an ASSERT_EQ on the Shape of ao?  If there is a code error and ao had a larger shape than tested by the double-for loop below, it wouldn't be caught.